### PR TITLE
CI/CD

### PR DESCRIPTION
**Pull request by MrCheater**

### CODE REVIEW

File: packages/create-resolve-app/bin/index.js [hunk 1]
```diff
@@ -3,9 +3,13 @@
 const commandLineArgs = require('command-line-args');
 const moduleCreator = require('../dist/create_resolve_app');
 
-const optionDefinitions = [{ name: 'scripts', type: String }, { name: 'sample', type: Boolean }];
+const optionDefinitions = [
+  { name: 'scripts', type: String },
+  { name: 'sample', type: Boolean },
+  { name: 'version', alias: 'v', type: String }
```
Comment (Vasiliy-Lavrov) [inline packages/create-resolve-app/bin/index.js]: Please use 'v' alias in the uppercase, sample code:
 { name: 'version', alias: 'V', type: String }

Comment (IhostVlad) [reply packages/create-resolve-app/bin/index.js]: Maybe `version` ia not good name for that purpose, but this `version` is not about CRA version, but about actial NPM published alpha for `resolve-scripts`

Comment (IhostVlad) [reply packages/create-resolve-app/bin/index.js]: In current context `version` is like `scripts` field, but accepts NPM version istead of path to target package

Comment (IhostVlad) [reply packages/create-resolve-app/bin/index.js]: Also, this PR is only for CI/CD R&D purposes, it will be removed after Jenkins will configured well

**Pull request closed without merging**